File: beer_bottles/python/beer_bottles.py
#!/usr/local/bin/python

for beer in xrange(99, 0, -1):
	print "%s bottles of beer on the wall" % str(beer)
	print "%s bottles of beer" % str(beer)
	print "Take one down, pass it around"
	print "%s bottles of beer on the wall" % str(beer-1)
	print ""

File: beer_bottles/java/beer_bottles.java
import java.util.Scanner;

public class beer_bottles {
	public static void main(String[] args) {
		for (int i=99; i>0; i--) {
			System.out.print(i+" bottles of beer on the wall\n"+i+" bottles of beer\nTake one down, pass it around\n"+(i-1)+" bottles of beer on the wall\r\n\r\n");
		}
	}
}